app/src/App.css

```diff
@@ -144,6 +143,67 @@ body::-webkit-scrollbar-thumb {
   color: #2b9e91;
 }
 
+.calendar .navBar {
+  display: grid;
+  box-sizing: border-box;
+  position: relative;
+  top: 0em;
+  left: 0em;
+  margin: auto;
+  padding: 0em 0.125em;
+  width: 100%;
+  height: auto;
+
+  grid-template-columns: 1fr 3fr 1fr;
```

Comment:
Does this grid template layout mean you are using a grid CSS template from a 3rd party library?

Reply:
Nope! Just css from scratch. `fr` is just another unit of measurement that stands for _fluid ratio_. So it’s not flexible, but not based on percentages or surrounding elements or viewport, it’s just about maintaining the ratio between the elements in those columns. Super nifty.